### PR TITLE
Addresses #5694

### PR DESCRIPTION
**Pull request by ignatandrei**
Microsoft.AspNetCore.StaticFiles": "1.1.0"
to project,json
Add
app.UseStaticFiles();
in the
public void Configure(IApplicationBuilder app)
from Startup class.

Addresses #5694

### CODE REVIEW

File: test/WebSites/HtmlGenerationWebSite/Startup.cs [hunk 1]
```diff
@@ -41,6 +41,7 @@ public void Configure(IApplicationBuilder app)
                     template: "{controller}/{action}/{id?}",
                     defaults: new { controller = "HtmlGeneration_Home", action = "Index" });
             });
+            app.UseStaticFiles();
```
Comment (Eilon) [inline test/WebSites/HtmlGenerationWebSite/Startup.cs]: Generally UseStaticFiles goes before UseMvc.

File: test/WebSites/HtmlGenerationWebSite/project.json [hunk 2]
```diff
@@ -13,7 +13,8 @@
       "version": "1.1.0-preview4-final"
     },
     "Microsoft.AspNetCore.Server.IISIntegration": "1.2.0-*",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.2.0-*"
+    "Microsoft.AspNetCore.Server.Kestrel": "1.2.0-*",
+    "Microsoft.AspNetCore.StaticFiles": "1.1.0"
```
Comment (Eilon) [inline test/WebSites/HtmlGenerationWebSite/project.json]: Use `1.2.0-*` for the version.

**Comment by Eilon** [comment]
@ignatandrei thanks for the PR! Can you quickly sign the CLA and address the review feedback so we can look into merging?

**Comment by ignatandrei** [comment]
I am getting now the approval from my company ( I think it will be 3-4 days from now) . If you need faster , then please reject the pull request and I will re-submit after I get the approval.
Sorry for that 
Andrei

**Comment by Eilon** [comment]
@ignatandrei no worries, a few days is no problem at all.

**Comment by Eilon** [comment]
(If you'd like, you can still send us an update to the PR in the meantime.)

**Comment by ignatandrei** [comment]
I have signed the document, I have also pushed to my fork .

**Pull request merged**